metrics/docs/02_sign_and_notarize_metrics_binaries.md

```diff
@@ -27,15 +27,27 @@ The document's goal is to investigate and analyze tools and methods that can hel
 - [generate sha256 hash](https://ss64.com/osx/shasum.html)
 - [import codesign certificate Github Action](https://github.com/Apple-Actions/import-codesign-certs)
 
-# Analysis
+## Contents
+
+- [**Analysis**](#analysis)
+    - [Landscape](#landscape)
+        - [gon](#gon)
+        - [signing_tools](#signing_tools)
+        - [notarize-cli](#notarize-cli)
+        - [Generate sha256 hash](#generate-sha256-hash)
+        - [Decision](#decision)
 
-## Process
+# Analysis
+> Describe a general analysis approach.
 
 The analysis begins with the selection of programs and the study of their work, complexity, and popularity. This provides a comprehensive view of how to solve the given problem.
 
 The research concludes with a short description of the tool (method), an example of the usage, and a list of pros and cons of each.
 
-### [gon](https://github.com/mitchellh/gon)
+### Landscape
+> Look for existing solutions in the area.
+
```

Comment:
:tw: I believe it would be great to have an opening sentence here. See [Google Technical Writing](https://developers.google.com/tech-writing/one/paragraphs#write_a_great_opening_sentence) for more details.